playbooks/support/README.md

```diff
@@ -35,6 +35,10 @@ with:
 These are different than non-urgent bugs/change requests/features, which are prioritized and tackled in teams'
 sprints. See [below](#examples) for some common examples.
 
+## Wait!
+
+If you are new to on-call or looking for a refresher, we recommend watching the [on-call onboarding session](https://www.dropbox.com/home/Artsy%20Engineering/Onboarding/On-Call) 🔒 we ran on 3/8/19 which talks through your responsibilities as an on-call engineer.
```

Comment:
Suggest changing "3/8/19" to "March 8, 2019" so that it doesn't get confused with August 3rd.

Reply:
Updated!